c/tests/testlib.c

```diff
@@ -48,7 +48,21 @@ const char *single_tree_ex_mutations
       "2    2     1   -1\n"
       "2    3     1   -1\n";
 
-/* Example from the PLOS paper */
+/*** Example from the PLOS paper ***/
```

Comment:
Fantastic! Thanks for putting these in, that's a major bonus.

Reply:
No probs. I just need to sort out the C linting (not set up on my system) and Ben's last comment, and this should be done, ready for 0.3.1.

Reply:
I can c lint if needed.

Reply:
Thanks @benjeffery - it would save me quite a lot of hassle if you could lint the C, so please do if that's OK. I've just pushed the commit with the "options" parameter, which is much better (thanks for the tip). I can also squash the commits into 1, or I guess you can, it it's easy (the first one, f9a4a972d346862c5ba22e4673b856a149256c01) is a separate PR, though, so perhaps leave that)

Reply:
Done, force pushed.

Reply:
Thanks a lot, Ben!